snapshot.yaml

```diff
@@ -4,10 +4,10 @@ packages:
   - AesonBson-0.4.0
   - csv-conduit-0.7.2.0
   - direct-daemonize-3.1
+  - esqueleto-3.4.0.1
```

Comment:
This is already in the snapshot: https://www.stackage.org/nightly-2020-11-28/package/esqueleto-3.4.0.1